### PR TITLE
Migration - eMove Case Study

### LINKED ISSUE #1790: 📄 Case Study - eMove
Migrate - URL: https://www.ssw.com.au/SSW/Consulting/Case-Study/eMove.aspx

```[tasklist]
### Tasks
- [x] Migrate the content
- [x] Add redirects in Front Door
```

### PR DESCRIPTION
**Pull request by jeoffreyfischer**
Migrated eMove case study

- Affected routes: /company/clients/emove

- Fixed #1790

- [ ] Include done video or screenshots

### CODE REVIEW

**Comment by jeoffreyfischer** [comment]
As per my conversation with @tiagov8, the two images carousel can be replaced with two figures next to each other (or under each other to allow bigger images).

File: content/company/clients.mdx [hunk 1]
```diff
@@ -568,7 +568,7 @@ _body:
           solution  and bug fixing existing errors on their existing site. SSW
           was also  commissioned to take a new design from eMove and create and
           implement  a DotNetNuke skin for the new design.
-        caseStudyUrl: 'https://www.ssw.com.au/SSW/Consulting/Case-Study/eMove.aspx'
+        caseStudyUrl: '/company/clients/e-move'
```
Comment (amankumarrr) [inline content/company/clients.mdx]: I think route should be `emove`, it's a single word

Comment (jeoffreyfischer) [reply content/company/clients.mdx]: Done

**Review by amankumarrr — APPROVED** [review]
LGTM 🥂

**Pull request merged**